Dockerfile

```diff
@@ -1,19 +1,47 @@
-FROM node:14 as builder
+FROM node:14-alpine as builder
 WORKDIR /app
 COPY package*.json ./
-RUN npm install
-COPY . .
-ARG APP_MOUNT_URI
+RUN npm ci
+
+
+COPY nginx/ nginx/
+COPY assets/ assets/
+COPY locale/ locale/
+COPY testUtils testUtils/
```

Comment:
Is `testUtils` actually needed? Sounds like it's something about a test suite?

Reply:
It is required to build Docker - without `testUtils` Docker build yields an error:
```#24 78.95 TS2307: Cannot find module '../../testUtils/ApolloMockedProvider' or its corresponding type declarations.```

Reply:
Maybe it's worth investigating why production code tries to import something designated for testing.

Reply:
yeah, let's investigate that, mocks should not be mandatory for building..

Reply:
sure let's take this as a next thing and close the docker/building topic